Source/Core/Core/IOS/USB/Bluetooth/BTReal.h

```diff
@@ -65,10 +65,10 @@ class BluetoothReal final : public BluetoothBase
 
 private:
   static constexpr u8 INTERFACE = 0x00;
-  // Arbitrarily chosen value that allows emulated software to send commands often enough
-  // so that the sync button event is triggered at least every 200ms.
+  // Arbitrarily chosen value that allows emulated software to not block if libusb transfers
```

Comment:
As the comment indicates, this value must be small enough to allow the sync button event to be triggered within a reasonable amount of time when the user decides to "sync" Wii remotes. This change will have a nasty side effect: now sometimes the event won't be delivered before 5 seconds, which is unacceptable for UX.

Btw, this likely won't do what you want for Bluetooth passthrough. Official titles have a timeout mechanism for USB transfers. If they don't complete within the expected amount of time, then they'll drop any late reply and stop submitting/processing transfers for a while. BT passthrough already works pretty badly with bad adapters that take too long to reply, *locally*; if you add network latency to this, it's safe to assume it will probably never work well at all with any adapter, so please revert this change.